release/build.xml

```diff
@@ -45,7 +45,7 @@
        Result assigned to: restcomm-rvd.download.url
   -->
   <property name="restcomm-rvd.path" value="${configuration.directory}/restcomm-rvd.war"/>
-  <condition property="restcomm-rvd.download.url" value="https://oss.sonatype.org/content/groups/public/org/restcomm/restcomm-connect-rvd/${rvd.version}/restcomm-connect-rvd-${rvd.version}.war" else="https://mobicents.ci.cloudbees.com/view/VisualDesigner/job/VisualDesigner/lastSuccessfulBuild/artifact/designer/target/restcomm-rvd.war">
+  <condition property="restcomm-rvd.download.url" value="https://oss.sonatype.org/content/groups/public/org/restcomm/restcomm-connect-rvd/${rvd.version}/restcomm-connect-rvd-${rvd.version}.war" else="https://cxs.restcomm.com/view/Upstream/job/UPS_VisualDesigner/lastSuccessfulBuild/artifact/designer/target/restcomm-rvd.war">
```

Comment:
better not to use views in the url, but is small detail..

is the script ready to authenticate against CXS to get the file?

Reply:
Ended up using github releases instead of cxs